packages/data-context/src/actions/WizardActions.ts

```diff
@@ -72,6 +72,14 @@ export class WizardActions {
     return this.ctx.coreData.wizard
   }
 
+  setErroredFrameworks (erroredFrameworks: Cypress.ErroredFramework[]) {
```

Comment:
TODO: call this from packages/scaffold-config/src/ct-detect-third-party.ts once errored frameworks have been collected